### ISSUE TITLE
Suggestions for SHOW STATS

### ISSUE THREAD

**Issue by randomJoe211** [vesoft-inc/nebula-graph#1143]
1. SUBMIT JOB STATS and SHOW STATS is a combined workflow, instead of running them separately, why not make them one statement?
2. If a user has never run SUBMIT JOB STATS but tries to run SHOW STATS directly, the error `[ERROR (-8)]: Job is not finished!` is returned. This error message is confusing. Maybe it should say something like "To make statistics of the graph space, run SUBMIT JOB STATS first, and then run SHOW STATS." And if suggestion 1 is adapted, this issue would never occur.
![image](https://user-images.githubusercontent.com/69501902/122187531-b116bf00-cec1-11eb-86e1-4f5fe984b25e.png)

**Comment by randomJoe211** [comment]
> for question 1,
> `SUBMIT JOB STATS` statement starts a job that makes the statistics of the CURRENT graph space.
> `SHOW STATS` means show the preview job statistics.
> 
> if make them one statement, it would alway start a new job, it's not comfortable for big data.

That makes sense. Thank you for the explanation. Then there is only question 2 left.

**Comment by panda-sheep** [comment]
Good catch, the problem of error message, I will improve it later, thank you
